service/service.go

```diff
@@ -315,3 +317,41 @@ func FindUnitServiceNames(svcNames []string) map[string]string {
 	}
 	return unitServices
 }
+
+// FindAgents finds all the agents available on the machine.
+func FindAgents(dataDir string) (string, []string, []string, error) {
```

Comment:
Is this just a move?

Reply:
Yes, this was part of service. systemdServiceManager but I yanked it out and exported it as it is not systemd-specific so I could use it with the migration step.

Reply:
Excellent. That file is in the wrong place (it's systemd specific), but relocation requires addressing circular deps, which is why it hangs about. Anything that makes it smaller is a plus.